website/docs/intro/install/alpine.mdx

```diff
@@ -0,0 +1,26 @@
+---
+sidebar_position: 1
+sidebar_label: Alpine Linux
+description: |-
+Install OpenTofu on Alpine Linux.
+---
+
+# Installing OpenTofu on Alpine Linux
+
+OpenTofu is available in the [Alpine Linux testing repository](https://pkgs.alpinelinux.org/packages?name=opentofu) or as an `.apk` package from the [GitHub releases page](https://github.com/opentofu/opentofu/releases/latest/).
+
+## Installing from the testing repository
+
+OpenTofu is available in the Alpine Testing repository and can be installed from there:
```

Comment:
**(nitpick, non-blocking)** Hey Jánosh, I find few sentences a bit repetitive, WDYT about the following:

```suggestion
## Installing from the testing repository

OpenTofu is available in the [Alpine Testing repository](https://pkgs.alpinelinux.org/packages?name=opentofu) and can be installed from there:
```

Reply:
Thank you for sharing the illustrations! I didn't think of unfortunate css complexity..

I'd suggest to go with your original proposal as it feels to require least effort for most value 😉

Reply:
Sounds good to me, I'll resolve this conversation then. If it ever comes up again, I'll give the admonition styling a do over to make it look better when it has embedded paragraphs. Thanks for the input!